docs/environment.yml

```diff
@@ -1,12 +0,0 @@
-name: jup_client
-channels:
-  - conda-forge
-dependencies:
-  - pyzmq
-  - python==3.7
-  - traitlets>=4.1
-  - jupyter_core
-  - sphinx>=1.3.6
-  - sphinx_rtd_theme
-  - pip:
-      - sphinxcontrib_github_alt
```

Comment:
I think the goal of that one is to install the compiled dependencies on readthedocs (pyzmq fails to build otherwise), but we can test see if that works.

Reply:
Here #521 RTD configuration based on conda environment was replaced with pip based. So it looks like it is not used anymore.